### PR TITLE
AI-90: Fix contact record pledges tab

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
**Before**

pledges-issues

**After**

resolved-pledges-issues

**Note:** Some issues are already resolved by other task and deployed.

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss [hunk 1]
```diff
@@ -120,7 +120,7 @@ $button-border-radius: 2px;
   .crm-clear-link {
     border: 0 !important;
     color: $gray-dark;
-    margin: 7px 0px !important;
+    margin: 0px !important;
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss]: Remove px when u use 0

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_pledges.scss [hunk 2]
```diff
@@ -1,6 +1,33 @@
 #Search.CRM_Pledge_Form_Search {
   table.selector {
     @include mx-expandable-table();
+
+    td {
+      .crm-hover-button:hover {
```
Comment (meladawy) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_pledges.scss]: can u minimize the hierarchy by doing td .crm-hover-button:hover

File: org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss]: Fixed.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_pledges.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_pledges.scss]: Fixed.

**Pull request merged**